### PR TITLE
docs(ke/1.3/plugins) GraphQL Quickstart

### PR DESCRIPTION
**Pull request by h3xar0n**
Before PR review:
- [x] add links
- [x] test links locally or on netlify staging
- [x] add navigation (Plugins > GraphQL > above all the other GraphQL plugin pages)

### CODE REVIEW

File: app/enterprise/1.3-x/plugins/graphql-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Getting Started with GraphQL and Kong Enterprise
+---
+
+GraphQL decouples apps from services by introducing a flexible query language. Instead of a custom API for each screen, app developers describe the data they need, service developers describe what they can supply, and GraphQL automatically matches the two together. Teams ship faster across more platforms, with new levels of visibility and control over the use of their data. To learn more about how teams benefit, read why [GraphQL is important](https://www.apollographql.com/why-graphql/).
+
+Kong Enterprise is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. Kong Enterprise quickly and consistently extends the features of your APIs. Some of the popular features deployed through Kong Enterprise include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about the benefits of Kong, please see the FAQ.
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/plugins/graphql-quickstart.md]:
```suggestion
Kong Enterprise is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. Kong Enterprise quickly and consistently extends the features of your APIs. Some of the popular features deployed through Kong Enterprise include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about these features, see the [Hub page](https://docs.konghq.com/hub/) for plugins. For more about the benefits of Kong in general, please see the [FAQ](https://konghq.com/faqs/).
```

File: app/enterprise/1.3-x/plugins/graphql-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+$ curl -i -X POST http://kong:8001/services/example-service/plugins \
+  --data name=graphql-rate-limiting-advanced \
+  --data config.limit=100,10000 \
+  --data config.window_size=60,3600 \
+  --data config.sync_rate=10
+```
+
+The GraphQL Rate Limiting Advanced plugin supports two rate-limiting strategies. The default strategy will try to estimate cost on queries by counting the nesting of nodes. The default strategy is meant as a good middle ground for general GraphQL queries, where it's difficult to assert a clear cost strategy, so every operation has a cost of 1.
+
+A more advanced strategy is available for GraphQL schemas that enforce quantifier arguments on any connection, providing a good approximation on the number of nodes visited for satisfying a query. Any query without decorated quantifiers has a cost of 1. It is roughly based on GitHub's GraphQL resource limits.
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/plugins/graphql-quickstart.md]:
```suggestion
A more advanced strategy is available for GraphQL schemas that enforce quantifier arguments on any connection, providing a good approximation on the number of nodes visited for satisfying a query. Any query without decorated quantifiers has a cost of 1. It is roughly based on [GitHub's GraphQL resource limits](https://developer.github.com/v4/guides/resource-limitations/).
```

File: app/enterprise/1.3-x/plugins/graphql-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Read more about rate-limiting here: [GraphQL Rate Limiting Advanced Plugin](/enterprise/{page.kong_version}/plugins/
+
+### New upstream
+
+We have prepared a quickstart guide that will help you build your new GraphQL service on top of Kong and Apollo.
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/plugins/graphql-quickstart.md]: I'm assuming we'll want to also publish this guide in docs, so in the future we'll change this to a docs link, right @eskerda and @GayleNeumann ?

```suggestion
We have prepared a [quickstart guide](https://github.com/Kong/kong-apollo-quickstart) that will help you build your new GraphQL service on top of Kong and Apollo.
```

File: app/_data/docs_nav_ee_1.3-x.yml [hunk 4]
```diff
@@ -277,6 +277,8 @@
           url: /plugins/oidc-auth0
     - text: GraphQL
       items:
+        - text: GraphQL Quickstart
```
Comment (h3xar0n) [inline app/_data/docs_nav_ee_1.3-x.yml]: @GayleNeumann is this title okay in the navigation? On the page itself, the title is "Getting Started with GraphQL and Kong Enterprise"

File: app/enterprise/1.3-x/plugins/graphql-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  --data config.limit=100,10000 \
+  --data config.window_size=60,3600 \
+  --data config.sync_rate=10
+```
+
+The GraphQL Rate Limiting Advanced plugin supports two rate-limiting strategies. The default strategy will try to estimate cost on queries by counting the nesting of nodes. The default strategy is meant as a good middle ground for general GraphQL queries, where it's difficult to assert a clear cost strategy, so every operation has a cost of 1.
+
+A more advanced strategy is available for GraphQL schemas that enforce quantifier arguments on any connection, providing a good approximation on the number of nodes visited for satisfying a query. Any query without decorated quantifiers has a cost of 1. It is roughly based on [GitHub's GraphQL resource limits](https://developer.github.com/v4/guides/resource-limitations/).
+
+Read more about rate-limiting here: [GraphQL Rate Limiting Advanced Plugin](/enterprise/{page.kong_version}/plugins/
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/plugins/graphql-quickstart.md]:
```suggestion
Read more about rate-limiting here: [GraphQL Rate Limiting Advanced Plugin](/enterprise/{page.kong_version}/plugins/graphql-rate-limiting-advanced)
```

File: app/enterprise/1.3-x/plugins/graphql-quickstart.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+  --data config.limit=100,10000 \
+  --data config.window_size=60,3600 \
+  --data config.sync_rate=10
+```
+
+The GraphQL Rate Limiting Advanced plugin supports two rate-limiting strategies. The default strategy will try to estimate cost on queries by counting the nesting of nodes. The default strategy is meant as a good middle ground for general GraphQL queries, where it's difficult to assert a clear cost strategy, so every operation has a cost of 1.
+
+A more advanced strategy is available for GraphQL schemas that enforce quantifier arguments on any connection, providing a good approximation on the number of nodes visited for satisfying a query. Any query without decorated quantifiers has a cost of 1. It is roughly based on [GitHub's GraphQL resource limits](https://developer.github.com/v4/guides/resource-limitations/).
+
+Read more about rate-limiting here: [GraphQL Rate Limiting Advanced Plugin](/enterprise/{page.kong_version}/plugins/graphql-rate-limiting-advanced)
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/plugins/graphql-quickstart.md]: interpolator needs double curlies.

```suggestion
Read more about rate-limiting here: [GraphQL Rate Limiting Advanced Plugin](/enterprise/{{page.kong_version}}/plugins/graphql-rate-limiting-advanced)
```

File: app/_data/docs_nav_ee_1.3-x.yml (hunk 4 above)
Comment (h3xar0n) [reply app/_data/docs_nav_ee_1.3-x.yml]: (Confirmed IRL that this works.)

**Comment by eskerda** [comment]
Didn't see this one. Thanks for merging the confluence page into docs!

**Pull request merged**